### ISSUE TITLE
[A11y] General - Link purpose - empty <a> tag

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1803]
### Operating System

Windows

### Browser

Edge

### Affected page

Homepage

### Admin user?

Yes

### Issue Description

There's an empty <a> tag used as the target for the skip link. This should probably just be a div or something. I don't know if it's a WCAG failure, it just confuses Accessibility Insights and might be a constant nuisance when testing things like this.

### Suggested Fix

Make the empty <a> tag a div or maybe even just make the main region be the target of the skip link

**Comment by CGillen** [comment]
Fixed by #1867
